### PR TITLE
K3s: Bump k3s v1.24.3 to v1.24.6

### PR DESCRIPTION
**Pull request by guptavinayak11**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

This PR bumps up the k3s v1.24.3 to v1.24.6 

###### Change Log  

Updated k3s to 1.24.6

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

Built locally with Mariner toolkit

~[Buddy-Build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305307)~

BuddyBuild: [PR-4798](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305938&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=1832)

### CODE REVIEW

File: SPECS/k3s/k3s-1.24.6.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Summary:        Lightweight Kubernetes
 Name:           k3s
-Version:        1.24.3
+Version:        1.24.6
 Release:        6%{?dist}
```
Comment (mfrw) [inline SPECS/k3s/k3s-1.24.6.spec:4]:
```suggestion
Release:        1%{?dist}
```

**Review by mfrw — COMMENTED** [review]
We missed to update the `Release`

**Comment by guptavinayak11** [comment]
> We missed to update the `Release`

Done. Thanks for pointing out.

**Review by mfrw — APPROVED** [review]
LGTM

**Pull request merged**